iocore/net/quic/QUICFrame.cc

```diff
@@ -63,6 +64,18 @@ QUICFrame::is_probing_frame() const
   return false;
 }
 
+void
+QUICFrame::set_owner(QUICFrameGenerator *owner)
```

Comment:
Do we need this?

Reply:
How about adding a parameter to the constructor and set the default value to nullptr? I'm worried about misuse of `set_owner()`. If we provided this function, `generated_by()` could return something else.

If we had to set it to something else, I would add `add_tracker(QUICFrameTracker* tracker)`, and `std::vector<QUICFrameTracker*> trackers()`, instead of `generated_by()`.